### PR TITLE
OpenVPN patchset update to find where Haiku publishes the driver

### PR DESCRIPTION
**Pull request by Swangeon**
Reflects change to where I publish the driver from [here](https://review.haiku-os.org/c/haiku/+/6608/31/src/add-ons/kernel/drivers/network/tun/driver.cpp#24)

### CODE REVIEW

File: net-vpn/openvpn/openvpn-2.6.4.recipe [hunk 1]
```diff
@@ -7,10 +7,13 @@ HOMEPAGE="https://openvpn.net"
 COPYRIGHT="2002-2023 OpenVPN Inc"
 LICENSE="OpenVPN
 	GNU GPL v2"
-REVISION="1"
+REVISION="3"
```
Comment (Begasus) [inline net-vpn/openvpn/openvpn-2.6.4.recipe:10]: Revision should go up one number, in this case it would need to be "2"

**Comment by Begasus** [comment]
Conflicting files. Did you create a new branch on a clean master or used your previous branch here?

File: net-vpn/openvpn/openvpn-2.6.4.recipe (hunk 1 above)
Comment (Begasus) [reply net-vpn/openvpn/openvpn-2.6.4.recipe:10]: Ah my bad, didn't look any further. :)

**Comment by Swangeon** [comment]
sorry about that, didn't realize that doing it on the same branch would be such a problem :p. Also thought I had a clan master but didn't. Thanks as always @OscarL & @Begasus!

**Pull request closed without merging**